app/views/admin/publishers/edit.html.slim

```diff
@@ -0,0 +1,15 @@
+#publisher
+  #publisher-section
+    - if flash[:notice]
+      .alert.alert-success role="alert"
+        = flash[:notice]
+    - if flash[:alert]
+      .alert.alert-danger role="alert"
+        = flash[:alert]
+    h2 = @publisher.name
+    = form_for(@publisher, url: admin_publisher_path, method: :patch) do |f|
+      .form-group
+        = f.label "Uphold widget disabled"
+        = f.check_box :excluded_from_payout, {}, checked: @publisher.excluded_from_payout?
+      .form-group
+        = f.submit
```

Comment:
Can we make this a button for consistency?
![image](https://user-images.githubusercontent.com/12549658/45124883-0ed72380-b13a-11e8-9d5f-92b86cfce1c1.png)

Reply:
`f.submit` is the standard convention for working with forms.

Reply:
sorry, I was referring to a bootstrap button